README.md

```diff
@@ -1,13 +1,15 @@
 # most dom-event
 
-Streamlined DOM events for [@most/core](https://github.com/mostjs/core).  Now you can write:
+Streamlined DOM events for [@most/core](https://github.com/mostjs/core). Now you can write:
 
 ```js
 import { click } from '@most/dom-event';
+import { tap runEffects } from "@most/core";
```

Comment:
Just noticed this needs a comma between `tap` and `runEffects`.

Reply:
shit man :man_facepalming: